libtransact/src/protocol/batch.rs

```diff
@@ -11,7 +11,7 @@ use crate::signing;
 
 use super::transaction::Transaction;
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Seems like this and the Debug addition are mixed up in commits.

Reply:
The Debug should've been in this commit (fixed); the PartialEq is necessary for checking existence of batch in queue